docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -1047,7 +1056,14 @@ Default stratifications to all observers should include scenario and input draw.
     - Check ENN mortality ratio compared to GBD and models 6.1-6.4
     - Mortality is overestimated to a degree greater than 6.3
     - `Notebook comparing model 6.4 to 6.1-6.4 <https://github.com/ihmeuw/vivarium_research_mncnh_portfolio/blob/main/verification_and_validation/model_6.1_through_6.4_nn_mortality_comparison.ipynb>`_
-
+  * - 6.5
+    - * Check that the neonatal mortality ratio is as expected in line with model 6.1
```

Comment:
It is time to start checking the cause-specific mortality rates as well!  Can you take a look?

Reply:
I just updated with a check on cause specific mortality. All causes are slightly underestimated as expected except for preterm birth which is slightly overestimated. I am not sure why this is but I made a note to check back on it and investigate.